### ISSUE TITLE
Change checklist names to reflect actual taxonomic range -- NEON Research Sites - *plants* 

### ISSUE THREAD

**Issue by kyule** [BioCache/NEON-Biorepository-Data-Docs#8]
**Describe the bug**
When you look at a specific taxon page it always says "Species within inventory project: NEON Research Sites - plants " no matter what the taxon is

**Comment by arbolitoloco** [comment]
@kyule has this been fixed? I can't reproduce this error.

**Comment by kyule** [comment]
No... I couldn't understand what I was talking about for a couple of minutes based on how I described the issue, but I found it again. I'm talking about getting something like this: 
<img width="389" alt="image" src="https://user-images.githubusercontent.com/4763928/65470327-9e23a780-de1f-11e9-98da-905901a6e3ff.png">

On this taxon page: **https://biorepo.neonscience.org/portal/taxa/index.php?taxon=92414**


Oncorhynchus is a genus of fish, but you will see "Species within inventory project: NEON Research Sites - plants " written above the thumbnails for the different species

**Comment by kyule** [comment]
Note that these inventories are  not seen on a species level taxon page (but are when you are looking at the genus or higher level)

Eg. Discoderus has this problem 
<img width="1138" alt="image" src="https://user-images.githubusercontent.com/4763928/75735814-ceb61a00-5cb8-11ea-8b86-ff51331ea253.png">
https://biorepo.neonscience.org/portal/taxa/index.php?tid=77334

but Discoderus acinopoides does not

<img width="1133" alt="image" src="https://user-images.githubusercontent.com/4763928/75735847-e392ad80-5cb8-11ea-8a25-2c7fab81f80e.png">
https://biorepo.neonscience.org/portal/taxa/index.php?tid=77335&taxauthid=1&clid=0
